### DISCUSSION TITLE
csvwriter: how to prevent escaping delimiter inside quotes?

### DISCUSSION THREAD

**Discussion by alexkleinwaechter** [JoshClose/CsvHelper#1374] [Q&A]
Hi Josh,
I'm on version 12.1.2.
My configuration is:
```
                    csv.Configuration.Delimiter = ",";
                    csv.Configuration.Quote = '"';
```
The resulting csv:
cbs_grp_g25,"Contruction\\, Site","Baustelle ""Grabehaufen"""

I don't want the escaped comma \\,
because the consuming side doesn't recognize it and saves the backslash.

Is there a configuration setting to switch off this behaviour?

Best regards,
Alex!

**Comment by alexkleinwaechter** [comment]
Closed issue by accident :-)

**Comment by alexkleinwaechter** [comment]
Has anybody an idea?

**Comment by AltruCoder** [comment]
CsvHelper should not be adding a backslash before commas.  Are you sure the data isn't coming to you that way?  The only other thing I can think of is some sort of encoding that is escaping the commas.

**Comment by JoshClose** [comment]
Can you create a small sample that reproduces this issue? As @AltruCoder said, CsvHelper doesn't do this by default.
